deps/build.jl

```diff
@@ -20,38 +23,61 @@ provides(BuildProcess,
          (@build_steps begin
           end),
          fftw)
-provides(BuildProcess,
+if !Sys.iswindows()
+    provides(BuildProcess,
          (@build_steps begin
           end),
          fftw_threads)
+end
 # ...because this does not work:
 #provides(Binaries, "usr", fftw)
 #provides(Binaries, "usr", fftw_threads)
 
-@BinDeps.install Dict(
-    :libfftw3 => :fftw,
-    :libfftw3_threads => :fftw_threads
-)
+if Sys.iswindows()
+    try
+        @BinDeps.install Dict(
+            :libfftw3 => :fftw
+        )
+    catch
+        # fftw3 has not been found
+        provides(Binaries, joinpath(pwd(),"usr","Library","bin"), fftw)
+        @BinDeps.install Dict(
+            :libfftw3 => :fftw
+        )
+    end
+else
+    @BinDeps.install Dict(
+        :libfftw3 => :fftw,
+        :libfftw3_threads => :fftw_threads
+    )
+end
 
 ## Now fftw is in place, download and build finufft
 libfinufft = library_dependency("libfinufft")
 
 provides(Sources,
-         URI("https://github.com/flatironinstitute/finufft/archive/1.1.1.zip"),
+        URI("https://github.com/flatironinstitute/finufft/archive/master.zip"),
```

Comment:
At least my workaround did not work on 1.1.1. Since the 1.1.1 release is already older than a year and a half, I suspect that there have been a few changes. I have to admit that I did not dig through them. Using the 1.1.1 branch, there are undefined references to the methods provided by FFTW.

Reply:
Ok, then we can stick with it for now, and fix the version next time there is a finufft release (hopefully soon)